applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -46,47 +58,12 @@ return view.extend({
 		o = s.taboption('general', form.Flag, "enabled", _("Enabled"));
 		o.rmempty = false;
 
-		o = s.taboption('general', form.Flag, "use_staging", _("Use staging server"),
-			_("Get certificate from the Letsencrypt staging server " +
-				"(use for testing; the certificate won't be valid)."));
-		o.rmempty = false;
-		o.modalonly = true;
-
-		o = s.taboption('general', form.ListValue, "keylength", _("Key size"),
-			_("Key size (and type) for the generated certificate."));
```

Comment:
Just removing this means that any old value set will just get lost when loading the configuration page (I think?). We should read the old value and convert it to the new one in this case, or we risk inadvertently breaking old configs...

Reply:
It won't, though: if both values are set, the old key_length takes precedence. So we have to take this into account somehow; since this is just a rename, I think the best UX would be if we can just load the old value, and if it's there, convert it to the corresponding key_type value, set that, and then remove the old value (doing this on load and only writing the new value if the user clicks save, of course).